dbt/include/athena/macros/materializations/models/table/create_table_as.sql

```diff
@@ -48,7 +48,7 @@
   with (
     table_type='{{ table_type }}',
     is_external={%- if table_type == 'iceberg' -%}false{%- else -%}true{%- endif %},
-  {%- if work_group_output_location is none -%}
+  {%- if table_type == 'iceberg' or work_group_output_location is none -%}
```

Comment:
what is the reason behind this? what if people using a specified `work_group_output_location ` and iceberg want to use the default workgroup location? 
Also, this seems a bit unrelated to the pull request, eventually, consider to move it into another PR ;) https://github.com/dbt-athena/dbt-athena/pull/202/files#r1164195147

Reply:
Right. I’ll split the PR next week. It’s a bit tricky though. The tests that I added are the basic test suite provided by dbt. All the changes I did were done in order to fix those tests. I could create a separate PR where I add the tests and include the fixes to make those tests work? I’d then skip the hive snapshot tests for now. Then the second PR would contain the changes related to the hive snapshots. Wdyt?

Reply:
And related to this comment: this actually gave me troubles. I had a workgroup with a default location, dbt failed when it tried to create a table, because the ‘location’ keyword was skipped. Apparently iceberg tables take the default location from the schema, not the athena workgroup.

Reply:
Ah gotcha, make sense as they are managed tables :)
Regarding splitting I'm fine to keep all the basic test suite in this one, as it could create less trouble.

Reply:
This should be covered by https://github.com/dbt-athena/dbt-athena/pull/231